test/util/invulnerables.ts

```diff
@@ -0,0 +1,24 @@
+import { DevModeContext } from "@moonwall/cli";
+
+export async function createBlockAndRemoveInvulnerables(context: DevModeContext) {
+    let nonce = (await context.polkadotJs().rpc.system.accountNextIndex(context.keyring.alice.address)).toNumber();
```

Comment:
Ideally we can change this to:
- first get the existing invulnerables
- then remove them

Here we are making the assumption that the invulnerables are alice, bob, charlie and dave but maybe it's worth changing

Reply:
An alternative could be to add an assert that after the create block, the list of invulnerables is empty. Then if in the future a test fails because of this, we will know it.